packages/onboarding/src/utils/flows.ts

```diff
@@ -128,6 +129,10 @@ export const isNewSiteMigrationFlow = ( flowName: string | null ) => {
 	return Boolean( flowName && [ SITE_MIGRATION_FLOW ].includes( flowName ) );
 };
 
+export const isSiteMigrationSignupFlow = ( flowName: string | null ) => {
```

Comment:
Nit: I would lean towards calling this `isMigrationSignupFlow()` to match the constant name.